### PR TITLE
Fix `implicit-function-declaration` warnings, pass 6

### PR DESCRIPTION
**Pull request by ribbanya**
Finishes the rest of `ft` and `gm`.

### CODE REVIEW

File: src/melee/ft/ftparts.c [hunk 1]
```diff
@@ -14,7 +14,9 @@
 #include <sysdolphin/baselib/pobj.h>
 #include <sysdolphin/baselib/util.h>
 
-#define MAX_FT_PARTS 140
+#include <dolphin/os/os.h>
+
+#define MAX_FT_PARTS (140)
```
Comment (r-burns) [inline src/melee/ft/ftparts.c:19]:
```suggestion
#define MAX_FT_PARTS 140
```
nit, parens not necessary for literals

**Review by r-burns — APPROVED** [review]

File: src/melee/ft/ftparts.c (hunk 1 above)
Comment (ribbanya) [reply src/melee/ft/ftparts.c:19]: Really? There are no adverse edge cases?

Comment (r-burns) [reply src/melee/ft/ftparts.c:19]: Nope, parens are only necessary when you are expanding an arg in a function-like macro or when you are expanding to multiple tokens

Comment (ribbanya) [reply src/melee/ft/ftparts.c:19]: Ok well I'm not motivated to fix it in bulk but I'll start phasing out the parens for literals.

**Pull request merged**